lib/util/ts_parser.js

```diff
@@ -684,6 +708,9 @@ shaka.util.TsParser = class {
       width: null,
       codec: null,
     };
+    if (this.videoCodec_ != 'avc') {
+      return videoInfo;
+    }
```

Comment:
Are we unable to get video info from HEVC?  Is that coming later in #5611?

Reply:
Yes, we can, but it will come in another PR, so we limit the scopes of the less important PRs and they are easier to review.